backend/src/Entity/TaskResult.php

```diff
@@ -33,4 +39,12 @@ public function __construct()
     {
         $this->createdAt = new \DateTimeImmutable();
     }
+
+    public static function fromOutput(string $output): self
+    {
+        $result = new self();
+        $result->output = $output;
+
+        return $result;
+    }
 }
```

Comment:
When creating a `TaskResult` using `fromOutput`, consider if you should also allow setting the agent and agentName properties for completeness.